src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -70,6 +70,14 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
                     response = new HttpResponseMessage(statusCode);
                     response.Content = new StringContent(body);
 
+                    // we default the Content-Type here, but we override below with any
+                    // Content-Type header the user might have set themselves
+                    if (Utility.IsJson(body))
```

Comment:
Functionally, it shouldn't make a difference, and perf wise, it should have minimal impact.